Номер3/Номер3/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Номер3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enterx = ");
```

Comment:
Опечатка. Нужен пробел
```suggestion
            Console.WriteLine("enter x = ");
```